evalai/utils/urls.py

```diff
@@ -1,12 +1,12 @@
 from enum import Enum
 
-
+GET_ALL_CHALLENGES_URL = "/api/challenges/challenge/time/{}/approved/{}/published/{}"
 class URLS(Enum):
     login = "/api/auth/login"
     get_access_token = "/api/accounts/user/get_auth_token"
-    challenge_list = "/api/challenges/challenge/all"
-    past_challenge_list = "/api/challenges/challenge/past"
-    future_challenge_list = "/api/challenges/challenge/future"
+    challenge_list = GET_ALL_CHALLENGES_URL.format("all", "all", "all")
+    past_challenge_list = GET_ALL_CHALLENGES_URL.format("past", "all", "all")
+    future_challenge_list =GET_ALL_CHALLENGES_URL.format("future", "all", "all")
```

Comment:
Any chance we can make the API call from old cli versions backward compatible with new URL we are moving to ?